### ISSUE TITLE
Existing project, Poetry init, check and install requirements

### ISSUE THREAD

**Issue by ST33LDI9ITAL** [sdispater/poetry#983]
- [ x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x ] I have searched the [documentation](https://poetry.eustace.io/docs/) and believe that my question is not covered.

## Feature Request

How about adding a check for existing requirements.txt to Poetry init so that if you init on an existing project it would auto install and add existing packages?

Basically integrate something like:
FOR /F "usebackq delims==" %i IN (`cat requirements.txt`) DO poetry add %i
for Windows, or
$ for item in $(cat requirements.txt); do   poetry add "${item}"; done
for Bash

**Comment by pmav99** [comment]
Α requirements file can contain all sort of stuff that cannot be parsed by something as simple as what you propose: https://pip.readthedocs.io/en/1.1/requirements.html

For the record, there was a PR for importing a pipenv project but sdispater chose to reject it: https://github.com/sdispater/poetry/pull/579

**Comment by hangtwenty** [comment]
Another issue link: https://github.com/sdispater/poetry/issues/663

**Comment by hangtwenty** [comment]
This one should be closed in favor of the existing tickets
